src/components/Map.vue

```diff
@@ -42,19 +42,30 @@ export default {
     syncMaps () {
       return this.$store.state.syncMaps
     },
-    // This wrapper allows access to the primary (left) map
-    // object so it can be provided to other components, but
+    // These wrappers allows access to the primary (left) map object
+    // and secondary (right) map object so it can be provided to other components, but
     // the object remains outside the scope of Vue and thus
     // won't get decorated with accessors, etc.
     primaryMapObject () {
       return maps.left.map
+    },
+    secondaryMapObject () {
+      return maps.right.map
     }
   },
   mounted () {
     // Instantiate map objects
     maps.left.map = this.$L.map('map-1', this.getBaseMapAndLayers())
     maps.right.map = this.$L.map('map-2', this.getBaseMapAndLayers())
 
+    // Put the (left) map center in the console to assist with coding
+    maps.left.map.on('zoomend', () => {
+      console.log('Map zoom: ', maps.left.map.getZoom())
```

Comment:
Leaving these "console.log"s in is sort-of deliberate but maybe should be handled differently.  The point is that it's exceedingly helpful, during development, to be able to see where the map is centered/zoom level.  Should these be `console.info` instead...?  Could maybe look for a debug flag...?

Potentially, too, we should just remove these and find the right path to the current lat/lon for a map object.  It's just harder to find there.